package.json

```diff
@@ -51,7 +51,7 @@
     "rollup": "^0.41.6",
     "standard-version": "^4.0.0",
     "tslint": "^5.1.0",
-    "typescript": "^2.3.2",
+    "typescript": "^2.4.0",
```

Comment:
There's no need to update typescript in library itself, at least for now till 2.4 is officially released